api/messaging/sms/sms_message_request.py

```diff
@@ -160,20 +164,21 @@ def send(self):
         requirements.
         """
 
-        if Validate.message(self.message) and Validate.number(self.to):
-            url = '{}&method={}&message={}&to={}&sender={}'.format(BASEURL, SMSMessageRequest.SMS, self.message,
-                                                                   self.to, SENDERID)
-            if self.dlr_url:
-                url += '&dlrurl={}'.format(self.dlr_url)
-            if self.custom:
-                url += '&custom={}'.format(self.custom)
-            if self.unicode:
-                url += '&unicode={}'.format(self.unicode)
-            if self.flash:
-                url += '&flash={}'.format(self.flash)
-            response = Klient(url).response()
-            sms_message_response = SMSMessageResponse(response=response)
-            return sms_message_response
+        if not Validate.message(self.message) or not Validate.number(self.to):
+            return
+
+        url = '{}&method={}&message={}&to={}&sender={}'.format(BASEURL, SMSMessageRequest.SMS, self.message,
+                                                               self.to, SENDERID)
+        if self.dlr_url:
+            url += '&dlrurl={}'.format(self.dlr_url)
+        if self.custom:
+            url += '&custom={}'.format(self.custom)
+        if self.unicode:
+            url += '&unicode={}'.format(self.unicode)
+        if self.flash:
+            url += '&flash={}'.format(self.flash)
+        response = Klient(url).response()
+        return SMSMessageResponse(response=response)
```

Comment:
Function SMSMessageRequest.send refactored with the following changes:

- Add guard clause ([last-if-guard](https://docs.sourcery.ai/refactorings/last-if-guard/))
- Inline variable that is immediately returned ([inline-immediately-returned-variable](https://docs.sourcery.ai/refactorings/inline-immediately-returned-variable/))